harmonica/forward/prism.py

```diff
@@ -73,6 +74,10 @@ def prism_gravity(
     dtype : data-type (optional)
         Data type assigned to the resulting gravitational field. Default to
         ``np.float64``.
+    progressbar : bool (optional)
+        If True a progress bar for the number of gravity coordinates to solve
+        in the forward calculation will be displayed using the numba-progress
+        package. Default to ``False``.
```

Comment:
Small changes in the docstrings to make it clearer how the progressbar will be printed.
BTW, we could add a link to `numba_progress` to the `intersphinx_mapping` dict in `doc/conf.py` so the `numba_progress` is clickable and links to its website.

```suggestion
        If True, a progress bar of the computation will be printed to standard error (stderr).
        Requires :mod:`numba_progress` to be installed.
        Default to ``False``.
```